<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_plot</class>
 <widget class="QMainWindow" name="win_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>161</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>161</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="login_lbl_welcome">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>204</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to Voice Trainer!</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_lbl_user">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QComboBox" name="login_cmb_users">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Users previously logged into the system</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn_cancel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn_ok">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn_register">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
